queue/abstract/driver/utube.lua

```diff
@@ -193,6 +449,19 @@ function method.kick(self, count)
         end
 
         task = self.space:update(task[1], {{ '=', 2, state.READY }})
+        if self.ready_space_mode then
+            local prev_task = self.space_ready_buffer.index.utube:get{task[3]}
+            if prev_task ~= nil then
+                delete_ready(self, prev_task[1], task[3])
+            else
```

Comment:
```suggestion
            if prev_task ~= nil then
                if prev_task[1] > task[1] then -- if id of the exist task > kicked task.
                     delete_ready(self, prev_task[1], task[3])
                end
            else
```

Reply:
Also did similar change for `utubettl` (but comparing pairs `{pri, id}`).